deploys/distributed/central/docker-compose-dev.yaml

```
@@ -58,6 +79,19 @@ services:
+  unoserver:
+    image: ghcr.io/eterna-earkiv/eterna-unoserver:latest
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD-SHELL", "nc -z localhost 2003 || exit 1"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 60s
+
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)